com.heroicgameslauncher.hgl.yml

```diff
@@ -0,0 +1,156 @@
+id: com.heroicgameslauncher.hgl
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+command: heroic-run
+separate-locales: false
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Compat.i386
+  - org.freedesktop.Sdk.Extension.toolchain-i386
+
+finish-args:
+  - --allow=devel
+  - --allow=multiarch
+  - --device=all
+  - --env=GST_PLUGIN_SYSTEM_PATH=/app/lib/gstreamer-1.0:/app/lib32/gstreamer-1.0:/usr/lib/extensions/gstreamer-1.0:/usr/lib/x86_64-linux-gnu/gstreamer-1.0:/usr/lib/i386-linux-gnu/gstreamer-1.0
+  - --filesystem=home
```

Comment:
Why does it need that permission?

Reply:
The default install folder is `$HOME/Games/Heroic`.

Reply:
Do you know how often users change this path? If it's a reasonable default, it would be great to limit it to this path and restore `--persist` line for Wine.

Reply:
What does persist does exactly?
I belive I can limit to a specific games folder then.
But probably lost users will install on other partitions but I've heard it's not ideal to add the host permission so I will let the users change the permissions with something like flatseal.
The thing is that the users can install games anywhere. Heroic will suggest the folder I said by default for both the games and wine prefixes.

Reply:
`--persist` maps given directory inside the sandbox to a "virtual home directory". It won't give access to `~/.wine` in the real home dir, but will cause anything written to `~/.wine` by the app to remain in place between app restarts.

Reply:
`--persist` will set up a symlink/bind mount to a location writable by default instead of exposing it from the host, so something like `--persist=.wine` for Wine and `--filesystem=~/Games/Heroic:create` (or maybe `--persist=Games`? it requires testing) should generally work. We tend to prefer limited permissions as they can be changed relatively easily, especially when you (as the app maintainer) control the default location of where files are stored, but if you insist on `--filesystem=home`, it's ultimately your decision.

Reply:
I see. Heroic can use Wine and Proton installed by non-flatpak versions of lutris and steam.
So it will need access for reading both folders inside home.
I can limit the permissions then, sounds more reasonable.

Reply:
I think `--filesystem=~/Games/Heroic:create` would be best. Lutris does the same:

https://github.com/flathub/net.lutris.Lutris/blob/323bab375bdb02e4fecafc280bd6e25df26a1d35/net.lutris.Lutris.yml#L21

Reply:
@TheEvilSkeleton changed that already on latest build 👍🏽